README.md

```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
 
   Checks all free-text fields in G-Cloud services for "bad words" and generates a CSV report of any bad words found.
 
+* `send_dos_opportunities_email.py`
```

Comment:
Ok, added to tech debt: https://trello.com/c/3Ca1I5m6/152-change-filename-of-senddosopportunitiesemailpy-in-scripts